dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
eclasses can change so you're not supposed to rely on what they inherit, just like we include dependencies when used directly by a package, even if another dependency already pulls it in.